bitstore/blueprint.py

```diff
@@ -1,13 +1,17 @@
 import json
+import os
 
 from flask import Blueprint, request, Response
-from . import controllers
 
+from . import controllers, services
 
 def make_blueprint():
     """Create blueprint.
     """
 
+    # Create FileManager tables if not exists
+    services.FileRegistry.init_db()
```

Comment:
I think it's better to do it in one place and not in two - to avoid collision.

Reply:
(that is - only create the db tables from specstore or here, but not both)

Reply:
it'll be only here